CONTRIBUTE

```diff
@@ -0,0 +1,69 @@
+CONTRIBUTING TO TQDM
```

Comment:
In fact it's included inside the `UPLOADING TO PYPI` chapter. I renamed this chapter to `BUILDING A RELEASE AND UPLOADING TO PYPI`. Is it ok now?